flixel/ui/FlxButton.hx

```diff
@@ -178,13 +178,20 @@ class FlxTypedButton<T:FlxSprite> extends FlxSprite implements IFlxInput
 	public var pressed(get, never):Bool;
 	public var justPressed(get, never):Bool;
 	
-	// we don't need an ID here, so let's just use Int as the type
+	/** 
+	 * We don't need an ID here, so let's just use Int as the type.
+	 */
 	private var input:FlxInput<Int>;
 	
 	/**
 	 * The input currently pressing this button, if none, it's null. Needed to check for its release.
 	 */
 	private var currentInput:IFlxInput;
+
+	/**
+	 * A helper for making sure animations won't trigger more than once per input.
+	 */
+	private var _status = -1;
```

Comment:
I was thinking about the far future, when someone who isn't aware of all the history would want to change it to a public var, but I guess we'll have to risk it :)